### DISCUSSION TITLE
403 Error?

### DISCUSSION THREAD

**Discussion by jasondukes** [fly-metothemoon/GWsearch#4] [General]
Is Frontier actively blocking the script now?

**Comment by neallutz** [comment]
I get the same error still. I'm guessing the URL path changed

**Comment by GQster** [comment]
I just ran it twice, once with the example call in the README 
`python gowild_scraper.py -o PHX -d 1 -t 1`
and once with my local airport.

Both worked. 
I tried running them again and for the 403 error
`0. Problem accessing URL: code 403`

**Reply by jasondukes** [reply]
It seems like the system is working most of the time again, but I still encounter 1 - 2 instances of error code 403 out of the 20 results. Upon checking the link that results in a 403 error, it appears to be a valid GoWild! flight, which is interesting. Additionally, it appears that the system is not checking every listing of the airport, which is unusual. It seems to only check for international flights based on the IATA codes displayed in the terminal.
